router/router.go

```diff
@@ -1994,6 +1993,31 @@ func (rt *HandleT) readAndProcess() int {
 			rt.timeGained += latenciesUsed[job.WorkspaceId]
 
 			rt.MultitenantI.CalculateSuccessFailureCounts(job.WorkspaceId, rt.destName, false, true)
+			/*
```

Comment:
I would replace this comment with what appears to be the standard comment markers for reporting, e.g.

```
//REPORTING - ROUTER - START

//REPORTING - ROUTER - END
```

Reply:
Done